client/src/App.jsx

```diff
@@ -1,17 +1,35 @@
 import { Route, Routes, NavLink } from 'react-router-dom';
+import { useState } from 'react';
 import Navbar from './components/Navbar';
 import Home from './pages/Home';
 import Create from './pages/Create';
 import Settings from './pages/Settings';
 import styled from 'styled-components';
+import { logData } from './components/LogData';
 
 function App() {
+  const [logs, setLogs] = useState(logData);
+
+  function AddLog(log) {
+    const addId = Object.keys(logs).length + 1;
```

Comment:
Why do you use `Object.keys()` here? Isn't `logs` an array?
```suggestion
    const addId = logs.length + 1;
```

Reply:
Nope it is an array filled with objects. Tried it with logs.length. But with no result.